.changeset/curvy-boxes-hide.md

```diff
@@ -0,0 +1,12 @@
+---
+"@logto/console": patch
+"@logto/phrases": patch
+---
+
+replace the i18n translated hook event label with the hook event value directly in the console
+
+- remove all the legacy interaction hook events i18n phrases
+- replace the translated label with the hook event value directly in the console
+  - `Create new account` -> `PostRegister`
+  - `Sign in` -> `PostSignIn`
+  - `Reset password` -> `PostResetPassword`
```

Comment:
should we keep the old name for display? e.g. `PostRegister (Create new account)`

Reply:
discussed offline. Add it in the docs instead.